add-on/src/options/forms/experiments-form.js

```diff
@@ -87,6 +90,17 @@ function experimentsForm ({
           </label>
           <input type="checkbox" id="ipfs-proxy" onchange=${onIpfsProxyChange} checked=${ipfsProxy} />
         </div>
+        ${ipfsNodeType === 'embedded' ? html`
+          <div>
+            <label for="ipfsNodeConfig">
```

Comment:
Now that we can safely recover any misconfiguration, lets move this back to the old place – below node type picker :)